Update storage-task-known-issues.md (#127058)
Azure/azure-content: articles/storage-actions/storage-tasks/storage-task-known-issues.md

The workaround is to exclude the specific prefixes which are soft deleted.

> ## No option to choose priority during re-hydration of blobs 

Reviewer: ```suggestion
## No option to choose priority when rehydrating blobs to an online tier 
```

> When re-hydrating archived storage blobs there is no option to choose priority. The blobs are re-hydrated with Standard priority. 

Reviewer: ```suggestion
When rehydrating archived blobs there is no option to choose priority. The blobs are rehydrated with standard priority. 
```

> When rehydrating archived blobs there is no option to choose priority. The blobs are rehydrated with standard priority. 

Reviewer 2: ```suggestion
When rehydrating archived blobs, there's no option to choose a priority. The blobs are rehydrated with the standard priority. 
```
